packages/nuxt/src/vite/sourceMaps.ts

```diff
@@ -164,16 +186,28 @@ export function getPluginOptions(
   }
 
   return {
-    org: sourceMapsUploadOptions.org ?? process.env.SENTRY_ORG,
-    project: sourceMapsUploadOptions.project ?? process.env.SENTRY_PROJECT,
-    authToken: sourceMapsUploadOptions.authToken ?? process.env.SENTRY_AUTH_TOKEN,
-    telemetry: sourceMapsUploadOptions.telemetry ?? true,
-    url: sourceMapsUploadOptions.url ?? process.env.SENTRY_URL,
+    // eslint-disable-next-line deprecation/deprecation
+    org: moduleOptions.org ?? sourceMapsUploadOptions.org ?? process.env.SENTRY_ORG,
+    // eslint-disable-next-line deprecation/deprecation
+    project: moduleOptions.project ?? sourceMapsUploadOptions.project ?? process.env.SENTRY_PROJECT,
+    // eslint-disable-next-line deprecation/deprecation
+    authToken: moduleOptions.authToken ?? sourceMapsUploadOptions.authToken ?? process.env.SENTRY_AUTH_TOKEN,
+    // eslint-disable-next-line deprecation/deprecation
+    telemetry: moduleOptions.telemetry ?? sourceMapsUploadOptions.telemetry ?? true,
+    // eslint-disable-next-line deprecation/deprecation
+    url: moduleOptions.sentryUrl ?? sourceMapsUploadOptions.url ?? process.env.SENTRY_URL,
+    headers: moduleOptions.headers,
     debug: moduleOptions.debug ?? false,
-    silent: sourceMapsUploadOptions.silent ?? false,
-    errorHandler: sourceMapsUploadOptions.errorHandler,
+    // eslint-disable-next-line deprecation/deprecation
+    silent: moduleOptions.silent ?? sourceMapsUploadOptions.silent ?? false,
+    // eslint-disable-next-line deprecation/deprecation
+    errorHandler: moduleOptions.errorHandler ?? sourceMapsUploadOptions.errorHandler,
+    bundleSizeOptimizations: moduleOptions.bundleSizeOptimizations, // todo: test if this can be overridden by the user
```

Comment:
l/just to confirm: Did we settle on the new build time options taking precedence over the old ones? I think this is generally fine (as fine as vice versa) but let's make sure we're consistent with this behaviour across sdks. (also doesn't hurt to write it down in the issue).

Reply:
Yeah we didn't specifically talk about that but I thought it makes sense as it makes migration easier as well. I'll write it down 👍